components/RecipeList.js

```diff
@@ -0,0 +1,52 @@
+import data from "@/db/data";
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function RecipeList() {
+  if (!data) {
+    return <p>Data not found</p>;
+  }
```

Comment:
I think it is safe to assume data exists seeing as we are directly importing it from a file.

Reply:
Hi Pete, yes you are absolutely right , we implemented it there because it is in our acceptance criteria and will be used when we connect our database. Should we remove it ?

Reply:
I would remove it for now.